front/app/containers/Admin/projects/project/participationContext/index.tsx

```diff
@@ -236,6 +246,12 @@ class ParticipationContext extends PureComponent<
     this.setState({ survey_embed_url });
   };
 
+  handleDocumentAnnotationEmbedUrlChange = (
```

Comment:
Minor: When we have a wrong url in the form. The error message isn't removed onChange. Might be a good improvement for later or now if you have the time

Reply:
Yes, error setting happens in parent and not happening anywhere in this file as far as I can see, so leaving that for now.